man/systemd.network.xml

```diff
@@ -809,6 +809,15 @@
             <para>As in the <literal>[Network]</literal> section.</para>
           </listitem>
         </varlistentry>
+         <varlistentry>
+           <term><varname>GatewayOnlink=</varname></term>
+           <listitem>
+             <para>Specifies that the kernel does not have to check if the gateway is reachable directly
+             by the current machine (i.e. the kernel does not need to check if the gateway is attached to
+             the local network). Explicitly consider the gateway to be onlink, even if there is no route
```

Comment:
Also, I still don't grok what this really does. What is the effective difference if I set this to true or false? Why would I see if I toggle this? What's the usecase for setting it?

I am just trying to grok what precisely this is about...